pom.xml

```diff
@@ -108,6 +108,15 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.6.0</version>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
```

Comment:
Why are we compiling at 1.8? 1.5 should be the minimum, so we can support projects with older versions of Java